tests/golden/defaults-with-provider-configs/crossplane/crossplane/70_provider_configs.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: helm.crossplane.io/v1beta1
+kind: ProviderConfig
+metadata:
+  annotations:
+    argocd.argoproj.io/sync-options: SkipDryRunOnMissingResource=true
+    argocd.argoproj.io/sync-wave: '10'
+  labels:
+    name: provider-name-one
+  name: provider-name-one
+spec:
+  credentials:
+    source: InjectedIdentity
+---
+apiVersion: pkg.crossplane.io/v1
+kind: ProviderConfig
```

Comment:
There is no `ProviderConfig` kind in the `pkg.crossplane.io` group. (https://doc.crds.dev/github.com/crossplane/crossplane)
Can we make the field `apiVersion` required so it doesn't default to `pkg.crossplane.io`?

Reply:
I was able to make the field mandatory, though I have no idea how to test it in golden tests.

Reply:
The manual tests have been done successfully.